scripts/detect-antigravity-installer.py

```
@@ -0,0 +1,191 @@
+
+        actual_sha256 = INSPECTOR.sha256_bytes(data)
+        parsed = urllib.parse.urlsplit(final_url) if installer_fixture is None else None
+        if installer_fixture is None:
+            if (
+                parsed is None
+                or parsed.scheme != "https"
+                or parsed.hostname != "antigravity.google"
+                or parsed.port not in (None, 443)
+                or parsed.username is not None
+                or parsed.password is not None
+            ):
```

Ruff (0.16.3)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)